backend/src/main/java/com/senior/project/backend/Activity/MilestoneHandler.java

```diff
@@ -10,12 +12,19 @@
 public class MilestoneHandler {
 
     private final MilestoneService milestoneService;
+    private final Logger logger = LoggerFactory.getLogger(MilestoneHandler.class);
 
     public MilestoneHandler(MilestoneService milestoneService){
         this.milestoneService = milestoneService;
     }
 
     public Mono<ServerResponse> all(ServerRequest serverRequest) {
-        return ServerResponse.ok().body(this.milestoneService.all(), Event.class );
+        var taskParam = serverRequest.queryParam("tasks");
+        if (taskParam.isPresent() && taskParam.get().equals("false")) {
```

Comment:
That has not been created yet. I made the get all milestones with/without tasks as an example, because the Lazy annotation can cause some confusing errors

Reply:
there is already a task api call so we can probably just copy the event implementation since it'll basically be the same. doesn't need to be implemented here tho as long as any task service can get all tasks in the database